BrainCloudClient/tests/TestEvent.cs

```diff
@@ -89,6 +89,53 @@ public void TestDeleteIncomingEvent()
 
             tr.Run();
         }
+        
+        [Test]
+        public void TestDeleteIncomingEvents()
+        {
+            TestResult tr = new TestResult(_bc);
+            
+            string eventId = SendDefaultMessage();
+            string[] eventIds = {eventId};
+            
+            _bc.EventService.DeleteIncomingEvents(
+                eventIds,
+                tr.ApiSuccess, tr.ApiError);
+
+            tr.Run();
+        }
+        
+        [Test]
+        public void DeleteIncomingEventsOlderThan()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            DateTime now = DateTime.UtcNow;
+            int dateInMillis = new DateTimeOffset(now).Millisecond;
+            
+            _bc.EventService.DeleteIncomingEventsOlderThan(
+                dateInMillis,
+                tr.ApiSuccess, tr.ApiError);
+
+            tr.Run();
```

Comment:
Just a comment for history:
it's not as complete as the C++ test where we create 2 events, then delete older than now + 2mins.
But only one of them needs to be fully functional. This one at least test the call

Reply:
I've made the change and it's been pushed. Please comment again if there's anything wrong with the test. I realize this is for history but I wanted to correct the tests.

Reply:
wait this shouldn't expect fail